scripts/bash/downloadEnvironmentVariables.sh

```diff
@@ -6,85 +6,85 @@ DIR=$(dirname "$0")
     # name (e.g. [deployment-name]-api.tupaia.org -> specific-deployment-api.tupaia.org)
-    sed -i -e "s/\[deployment-name\]/${DEPLOYMENT_NAME}/g" "${ENV_FILE_PATH}"
+    sed -i -e "s/\[deployment-name\]/${DEPLOYMENT_NAME}/g" "$ENV_FILE_PATH"
    
 
-    if [[ "${DEPLOYMENT_NAME}" == *-e2e || "${DEPLOYMENT_NAME}" == e2e ]]; then
+    if [[ "$DEPLOYMENT_NAME" == *-e2e || "$DEPLOYMENT_NAME" == e2e ]]; then
         # Update e2e environment variables
         if [[ ${FILE_NAME} == "aggregation" ]]; then
-            sed -i -e 's/^AGGREGATION_URL_PREFIX="?dev-"?$/AGGREGATION_URL_PREFIX=e2e-/g' ${ENV_FILE_PATH}
+            sed -i -e 's/^AGGREGATION_URL_PREFIX="?dev-"?$/AGGREGATION_URL_PREFIX=e2e-/g' "$ENV_FILE_PATH"
         fi
     fi
 
-    if [[ "${DEPLOYMENT_NAME}" == dev ]]; then
+    if [[ "$DEPLOYMENT_NAME" == dev ]]; then
         # Update dev specific environment variables
         # (removes ###DEV_ONLY### prefixes, leaving the key=value pair uncommented)
         # (after removing prefix, if there are duplicate keys, dotenv uses the last one in the file)
-        sed -i -e 's/^###DEV_ONLY###//g' ${ENV_FILE_PATH}
+        sed -i -e 's/^###DEV_ONLY###//g' "$ENV_FILE_PATH"
     fi
- 
 
-     echo "downloaded .env vars for $FILE_NAME"
+    echo -en "\033[2K\033[G" # Clear current line and set cursor to start of line
+    echo -e "\033[32m✅ Downloaded variables for \033[1m$FILE_NAME\033[m → $ENV_FILE_PATH"
 }
  
 for PACKAGE in $PACKAGES; do
-    # only download the env file if there is an example file in the package. If there isn't, this means it is a package that doesn't need env vars
-    has_example_env_in_package=$(find $DIR/../../packages/$PACKAGE -type f -name '*.env.example' | wc -l)
-    if [ $has_example_env_in_package -eq 1 ]; then
-        load_env_file_from_bw $PACKAGE $DIR/../../packages/$PACKAGE ""
+    # Only download the env file if there is an example file in the package. If there isn't, this means it is a package that doesn't need env vars
+    has_example_env_in_package=$(find "$DIR/../../packages/$PACKAGE" -type f -name '*.env.example' | wc -l)
+    if [ "$has_example_env_in_package" -eq 1 ]; then
+        load_env_file_from_bw "$PACKAGE" "$DIR/../../packages/$PACKAGE" ""
     fi
 done
  
 
-# get all .env.*.example files in the env directory
-file_names=$(find $DIR/../../env -type f -name '*.env.example' -exec basename {} \;)
- 
- 
-# for each file, get the extract the filename without the .example extension
+# Get all *.env.example files in the env directory
+file_names=$(find "$DIR/../../env" -type f -name '*.env.example' -exec basename {} \;)
+
+# For each file, get the base filename without the .env.example extension
 for file_name in $file_names; do
-    env_name=$(echo $file_name | sed 's/\.env.example//')
-    load_env_file_from_bw $env_name $DIR/../../env $env_name
+    env_name="${file_name%.env.example}"
```

Comment:
Pretty straightforward find/replace where `.env.example` is always the suffix, so [parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) makes a bit more sense